### PR TITLE
add initial noun translations

### PR DESCRIPTION
**Pull request by tatupesonen**

### CODE REVIEW

**Comment by tatupesonen** [comment]
@Eldemarkki please review

File: fi.json [hunk 1]
```diff
@@ -1720,23 +1720,23 @@
...
+        "link-with-count_plural": "{{count}} linkkiä",
+        "file-with-count": "{{count}} tiedosto",
+        "file-with-count_plural": "{{count}} tiedostoa",
+        "folder-with-count": "{{count}} kansio",
+        "folder-with-count_plural": "{{count}} kansiota",
+        "backlink-with-count": "{{count}} referoiva linkki",
+        "backlink-with-count_plural": "{{count}} referoivaa linkkiä",
+        "site-with-count": "{{count}} sijainti",
+        "site-with-count_plural": "{{count}} sijaintia",
+        "plugin-with-count": "{{count}} liitännäinen",
```
Comment (Eldemarkki) [inline fi.json]: I've used the word "lisäosa" for plugin and I think we should only use 1 translation for it. We should either change this to "lisäosa" or change the other places to use "liitännäinen". Both are fine in my opinion, you can decide which we should go with.

Comment (tatupesonen) [reply fi.json]: "Lisäosa" is shorter, I'll fix this PR up by today

**Comment by Eldemarkki** [comment]
Yep looks good!

**Pull request merged**